core/templates/pages/landing-pages/topic-landing-page/topic-landing-page.directive.html

```diff
@@ -1,190 +1,135 @@
 <div class="oppia-landing-page">
-  <div class="oppia-landing-section text-center" style="background-color: #afd2eb">
+  <div class="oppia-landing-section text-center oppia-landing-page-banner">
     <div class="oppia-landing-section-inner">
-      <div class="container-fluid">
-        <background-banner class="oppia-landing-background-image"></background-banner>
-        <div class="row oppia-landing-section-row">
-          <div class="col-sm-6 col-sm-push-6 oppia-landing-image-div">
-            <img ng-src="<[$ctrl.image1.src]>" class="oppia-landing-image" alt="<[$ctrl.image1.alt]>">
+      <div class="oppia-flex-box oppia-flex-row oppia-flex-mobile-column oppia-flex-mobile-align-center">
+        <div class="oppia-flex-item oppia-landing-page-image-container oppia-flex-mobile-order-2">
+          <img class="oppia-devices-image" ng-src="<[$ctrl.lessonInDevicesImageSrc]>" alt="Oppia lesson in different devices.">
```

Comment:
Could you please modify all the images to also support WebP format, the general approach how to do this is explained in #8499.

Reply:
Also, please make sure that the images are properly sized. They are big enough but not too much.

Reply:
Done!